homework/yaryna/palindrom_2method_hw_5.py

```diff
@@ -0,0 +1,15 @@
+string = "козак з казок"
+
+def palindrome(string):
+    string = string.lower().replace(" ","")
+    list_of_string = list(string)
+
+    list_of_string_2 = list(string)
+    list_of_string_2.reverse()
+
+    if list_of_string == list_of_string_2:
```

Comment:
make your function in one line :) 
*you do not need converting to list, because string also is able to do it